src/services/nnas/routes/account-settings.ts

```diff
@@ -0,0 +1,215 @@
+import crypto from 'node:crypto';
+import express from 'express';
+import got from 'got';
+import { z } from 'zod';
+import { getServerByClientID, getPNIDByPID } from '@/database';
+import { LOG_ERROR } from '@/logger';
+import { decryptToken, unpackToken, getValueFromHeaders, sendConfirmationEmail } from '@/util';
+import { config } from '@/config-manager';
+import { HydratedServerDocument } from '@/types/mongoose/server';
+import { HydratedPNIDDocument } from '@/types/mongoose/pnid';
+import { AccountSettings } from '@/types/services/nnas/account-settings';
+import { Token } from '@/types/common/token';
+import { RegionLanguages } from '@/types/services/nnas/region-languages';
+import { RegionTimezone, RegionTimezones } from '@/types/services/nnas/region-timezones';
+import { Country, Region } from '@/types/services/nnas/regions';
+import timezones from '@/services/nnas/timezones.json';
+import regionsList from '@/services/nnas/regions.json';
+
+const router: express.Router = express.Router();
+
+const accountSettingsSchema = z.object({
+	gender: z.enum(['M', 'F']),
+	tz_name: z.string(),
+	region: z.coerce.number(),
+	country: z.string(),
+	email: z.string().email(),
+	server_selection: z.enum(['prod', 'test', 'dev']),
+	marketing_flag: z.enum(['true', 'false']).transform((value) => value === 'true'),
+	off_device_flag: z.enum(['true', 'false']).transform((value) => value === 'true'),
+});
+
+/**
+ * [GET]
+ * Replacement for: https://account.nintendo.net/v1/account-settings/ui/profile
+ * Description: Serves the Nintendo Network ID Settings page for the Wii U
+ */
+router.get('/ui/profile', async function (request: express.Request, response: express.Response): Promise<void> {
+	const server: HydratedServerDocument | null = await getServerByClientID('3f3928cc6f780638d360f0485cef973f', config.server_environment);
+	const token: string | undefined = getValueFromHeaders(request.headers, 'x-nintendo-service-token');
+	if (!server || !token) {
+		response.sendStatus(504);
+		return;
+	}
+	const aes_key: string = server?.aes_key;
```

Comment:
Spacing between these

```suggestion
	const token: string | undefined = getValueFromHeaders(request.headers, 'x-nintendo-service-token');

	if (!server || !token) {
		response.sendStatus(504);
		return;
	}

	const aes_key: string = server?.aes_key;
```